doc/tutorials/content/sources/custom_executor/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+project(custom_executor)
+
+find_package(PCL 1.11.1.99 COMPONENTS common filters REQUIRED)
```

Comment:
1.11.1? Does CMake even work with 4 parts version?

Reply:
It does work `major[.minor[.patch[.tweak]]]`, from their docs.
Will switch to 1.11.1.

Reply:
That's technically incorrect. It places an expectation that the executor functionality is already available in 1.11.1 which is not the case. Setting the version to 1.x.x.99 is the common procedure when tutorials are added with features not yet released. 

It used to be part of our release tasks to make find replaces to the tutorials right before the releases (https://github.com/PointCloudLibrary/pcl/wiki/Preparing-Releases/247cd5b2c71bcc518b0c5eeea760b329954ff847 but apparently it was removed with the new release pipeline).